charts/vela-core/templates/defwithtemplate/deploy.yaml

```diff
@@ -21,15 +21,23 @@ spec:
         	policies:                 parameter.policies
         	parallelism:              parameter.parallelism
         	ignoreTerraformComponent: parameter.ignoreTerraformComponent
+        	inlinePolicies:           parameter.inlinePolicies
+        }
+        #Policy: {
+        	type:  string
+        	name?: string
+        	properties: {...}
```

Comment:
Will the properties be empty?

Reply:
> Will the properties be empty?

No, the policy type here needs to be one of "topology", "override", "replication". I have just added constraints on the policy type.